tests/resources/openldap/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+
+services:
+  openldap:
+    image: docker.io/bitnami/openldap:2.5
+    ports:
+      - '1389:1389'
+      - '1636:1636'
+    environment:
+      # https:#github.com/bitnami/bitnami-docker-openldap
+      LDAP_ROOT: "dc=example,dc=com"
+      LDAP_PORT_NUMBER: 1389
+      LDAP_ADMIN_USERNAME: "admin"
+      LDAP_ADMIN_PASSWORD: "insecure"
+      LDAP_ALLOW_ANON_BINDING: "yes"
+      LDAP_LOGLEVEL: 32
+      BITNAMI_DEBUG: "true"
+      LDAP_SKIP_DEFAULT_TREE: "yes"
+      LDAP_CONFIG_ADMIN_ENABLED: "yes"
+    volumes:
+      - './openldap:/bitnami/openldap'
+      - './ldif-extras:/ldif-extras'
+      - './ldif-extras/acl-mod.ldif:/ldifs/acl-mod.ldif'
+      - './ldif-extras/example.com.ldif:/ldifs/example.com.ldif'
+      - './ldif-fixtures/manager.example.com.ldif:/ldifs/manager.example.com.ldif'
+      - './ldif-fixtures/test.example.com.ldif:/ldifs/test.example.com.ldif'
+      - './ldif-fixtures/user1.example.com.ldif:/ldifs/user1.example.com.ldif'
+      #- './ldif-extras:/custom/ldif-extras'
+      #- './ldif-fixtures:/custom/ldif-fixtures'
```

Comment:
volumes mapping seem to be messed up. Could those be cleaned up, please?

Reply:
perhaps manual load of ldif files, utilize the `/docker-entrypoint-initdb.d` by the image:

- https://hub.docker.com/r/bitnami/openldap/#:~:text=Initializing%20a%20new%20instance

Reply:
Good point! I've unified the setup script but there are some limitations of `/docker-entrypoint-initdb.d` e.g. LDAP instance is stopped before running scripts from  `/docker-entrypoint-initdb.d`. More details can be found in `init.sh`

Reply:
I've checked the script you shared, perhaps `LDAP_CUSTOM_SCHEMA_DIR` can be just used for the load?
- https://github.com/bitnami/containers/blob/fccaa4c4a4d7755c19c2e02ddef7ac3736dfcbb9/bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh#L361-L376